### ISSUE TITLE
Unable to access NavigationEvents

### ISSUE THREAD

**Issue by lalithnarayan** [infinitered/ignite-andross#271]
System
platform darwin
arch x64
cpu 4 cores Intel(R) Core(TM) i5-5250U CPU @ 1.60GHz
directory /Users/lalithnarayankashyap/Documents/testApp

JavaScript
node 10.15.3 /usr/local/bin/node
npm 6.9.0 /usr/local/bin/npm
yarn 1.7.0 /usr/local/bin/yarn

React Native
react-native-cli 2.0.1
app rn version 0.57.7

Ignite
ignite 2.2.3 /usr/local/bin/ignite
boilerplate ignite-andross

Android
java 1.8.0_121 /usr/bin/java
android home - undefined

iOS
xcode 10.2.1

My Navigation looks like

```
export default createAppContainer(createSwitchNavigator(
  {
    AuthLoading: AuthLoadingScreen,
    App: MyDrawerNavigator,
    Auth: AuthStack,
  },
  {
    initialRouteName: 'AuthLoading',
  }
));
```
```
const MyDrawerNavigator = createDrawerNavigator({
  Home: {
    screen: home,
  },
},
  {
    contentComponent: DrawerComponent
  });

const AuthStack = createStackNavigator(
  {
    Login: LoginScreen,
  },
  {
    headerMode: 'none',
    initialRouteName: 'Login'
  }
);
```
this is my render
```

  render() {
    return (
      <Container>
        <NavigationEvents
          onWillFocus={payload => console.log('will focus', payload)}
          onDidFocus={payload => console.log('did focus', payload)}
          onWillBlur={payload => console.log('will blur', payload)}
          onDidBlur={payload => console.log('did blur', payload)}
        />
      </Container>

    )
  }
```

**Comment by superp** [comment]
I had the same problem, try this PR:
https://github.com/infinitered/ignite-andross/pull/273

**Comment by jamonholmgren** [comment]
Fixed by #273.
